circle/circle.go

```diff
@@ -17,6 +18,13 @@ Square: 201.06192982974676
 */
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
+	fmt.Println("enter radius")
+	var r float64
+	fmt.Scanln(&r)
+	p := r * 2 * math.Pi
+	s := r * r * math.Pi
+	fmt.Println("S=", s)
+	fmt.Println("P=", p)
 	// ++ Your code here! ++
```

Comment:
nit: We recommend to remove these comments. They were here as some kind of a placeholder.